library/nxos_switchport

```diff
@@ -72,17 +72,31 @@ options:
         aliases: []
     trunk_vlans:
         description:
-            - if mode=trunk, used as the vlan range to carry over trunk
+            - if mode=trunk and state=present:
+                - add vlans to trunk
+            - if mode=trunk and state=absent:
+                - remove vlans from trunk
+              
         required: false
         default: '1-4094'
         choices: []
         aliases: []
     state:
         description:
-            - Manage the state of the resource
+            - if state=present:
+                - if mode=access, set access port to vlan access_vlan
+                - if mode=trunk, set trunk port with native_vlan and
+                  add trunk_vlans as applicable
+            - if state=absent:
+                - if mode=access, remove access_vlan from access port
```

Comment:
To clarify, this is if mode=access, un-configure the existing access vlan.  If it's already vlan 1, there shouldn't be any change.